integration-tests/pom.xml

```diff
@@ -540,6 +540,15 @@
                 </includes-failsafe>
           </properties>
         </profile>
+        <profile>
```

Comment:
Can we define profile with tag definition as follow ...
```
<profile>
            <id>openshift</id>
            <properties>
                <skipITs>false</skipITs>
                <groups>openshift</groups>
            </properties>
        </profile>
```

Reply:
done

Reply:
done